account-integrations/sdk/src/accounts/SafeECDSAAccontWrapper.ts

```diff
@@ -0,0 +1,249 @@
+import { ethers } from 'ethers';
+import { z } from 'zod';
+import {
+  SafeECDSAPlugin,
+  SafeECDSAPlugin__factory,
+  SafeECDSARecoveryPlugin,
+  Safe__factory,
+  SafeECDSAFactory,
+} from '../../../../demos/inpage/hardhat/typechain-types';
```

Comment:
We will probably want to create a hardhat folder in the `sdk` directory so we don't have to link to the demo